Makefile

```diff
@@ -296,11 +207,11 @@ define os_error
 $(error Error: unsupported platform OS_TYPE:$1, OS_ARCH:$2; to mitigate this problem set variable KYMA with absolute path to kyma-cli binary compatible with your operating system and architecture)
 endef
 
-KYMA_FILE_NAME ?=  $(shell ./hack/get_kyma_file_name.sh ${OS_TYPE} ${OS_ARCH})
+KYMA_FILENAME ?=  $(shell ./hack/get-kyma-filename.sh ${OS_TYPE} ${OS_ARCH})
 KYMA_STABILITY ?= unstable
 
-kyma: $(LOCALBIN) $(KYMA) ## Download kyma locally if necessary.
+kyma: $(LOCALBIN) $(KYMA) ## Download Kyma cli locally if necessary.
```

Comment:
I think we should make this a `.PHONY` target since it checks for the existence of the Kyma binary itself. Please double-check.